python/pylibcudf/pylibcudf/tests/io/test_orc.py

```diff
@@ -52,3 +55,71 @@ def test_read_orc_basic(
     )
 
     assert_table_and_meta_eq(pa_table, res, check_field_nullability=False)
+
+
+@pytest.mark.skipif(
+    version.parse(pd.__version__) < version.parse("2.2.3"),
+    reason="fails on a pandas version < 2.2.3",
+)
+@pytest.mark.parametrize(
+    "compression",
+    [
+        plc.io.types.CompressionType.NONE,
+        plc.io.types.CompressionType.SNAPPY,
+    ],
+)
+@pytest.mark.parametrize(
+    "statistics",
+    [
+        plc.io.types.StatisticsFreq.STATISTICS_NONE,
+        plc.io.types.StatisticsFreq.STATISTICS_COLUMN,
+    ],
+)
+@pytest.mark.parametrize("stripe_size_bytes", [None, 65536])
+@pytest.mark.parametrize("stripe_size_rows", [None, 512])
+@pytest.mark.parametrize("row_index_stride", [None, 512])
+def test_roundtrip_pd_dataframe(
+    compression,
+    statistics,
+    stripe_size_bytes,
+    stripe_size_rows,
+    row_index_stride,
+    tmp_path,
+):
+    pdf = pd.DataFrame({"a": [1.0, 2.0, None], "b": [True, None, False]})
```

Comment:
Could you rewrite this test in terms of pyarrow so we don't have an implicit pandas dependency?

(FWIW pandas ORC operations just uses pyarrow anyways)

Reply:
Thanks, done.